src/reputation/lib.py

```diff
@@ -31,28 +634,25 @@ def _calculate_tokens_and_update_withdrawal_amount(self):
 
     def _request_transfer(self, token):
         url = ASYNC_SERVICE_HOST + '/ethereum/erc20transfer'
-        message_raw = {
+        message = {
             "balance_record": self.balance_record_id,
             "withdrawal": self.withdrawal.id,
             "token": token,
             "to": self.withdrawal.to_address,
             "amount": self.token_payout
         }
-        signature, message, public_key = ethereum.utils.sign_message(
-            message_raw,
-            WEB3_SHARED_SECRET
-        )
-        data = {
-            "signature": signature,
-            "message": message.hex(),
-            "public_key": public_key
-        }
-        response = http_request(
-            RequestMethods.POST,
-            url,
-            data=json.dumps(data),
-            timeout=3
+
+        contract = w3.eth.contract(abi=contract_abi, address=Web3.toChecksumAddress(RSC_CONTRACT_ADDRESS))
+        amount = int(self.amount)
```

Comment:
Should this be a decimal?

Reply:
we're only allowing users to put withdraw integer amounts at the moment